source/SSSC/sssc34.xml

```diff
@@ -4206,7 +4206,7 @@
 <w canon="ka-nipahitin" class="vta" gcat="1s2sTA" preverbs="ka|will" sgloss="kill" stem="nipah" suffix="-itin">ka-nipahitin</w>
 ,
 <w canon="êkâ" class="ipc" gcat="IPC" sgloss="not_to" stem="êkâ">êkâ</w>
-<w canon="kâ-wî-kâ-kitosiyan" class="vta" gcat="2s1sTAcnj" preverbs="kâ|cnj wî|will dupH|H.REDUP" sgloss="address" stem="kitot" suffix="-iyan">kâ-wê-kâ-kitosiyin</w>
```

Comment:
This I believe should be (ēkā) kāwi kā-kitosiyan (you will not address me back)

Reply:
Good call, I think it makes sense in context. I'll address this in a different PR